content/webapp/services/prismic/fetch/guides.ts

```diff
@@ -16,7 +16,7 @@ const guideFormatsFetcher = fetcher<GuideFormatPrismicDocument>(
 );
 
 type GuidesQueryProps = GetByTypeParams & {
-  format?: string | string[];
+  format?: string;
```

Comment:
This is because of the changes to `looksLikePrismicId` – previously we got `string | string[]` direct from the query parameter, but now we know the `format` parameter is a `string`, we can drop this.